build.gradle

```diff
@@ -4,13 +4,15 @@ buildscript {
     }
 
     dependencies {
+        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.1.3-2'
```

Comment:
let's make the version a variable so that also the jre gets updated once it's changed.